docs/4.4/api-reference.md

```diff
@@ -818,14 +820,99 @@ if err := client.DeleteAccessRequest(ctx, accessReqID); err != nil {
+
+You may want to use this API to manage your CA if:
+
+- You don't want to automate rotating certificates with `tctl` ([rotating CA with tctl](admin-guide.md#certificate-rotation))
+- You want to set up a custom system for rotating certificates for more security and stability
+- You want to rotate certificates in a trusted cluster without manually fetching the new certs on remote clusters
+
+### Certificate Rotation
+
+To maintain security across your cluster, it is a good idea to set up automatic [certificate rotation](architecture/authentication.md#certificate-rotation). The most fault tolerant, fast, and effective way to rotate the CA is to use a custom solution, but we'll start with the default automated solution and work from there.
+
+You can use `tctl auth rotate` or the RPC below to start the rotation in `auto` mode. You can also provide a custom grace period, or target only the host/user CA with both the RPC and `tctl auth rotate --type=user --grace-period=10h`. type is useful if you want to rotate the user and host CA's with different strategies.
+
+```go
+// This will start the an automatic rotation that schedules each phase of the rotation in equal increments.
+gracePeriod := time.Hour * 24
+req := auth.RotateRequest{
+  Mode: services.RotationModeAuto
+  GracePeriod: &gracePeriod // defaults to 48 hours
+  Type: services.UserCA // Leave empty to target UserCA and HostCA
+}
+if err := client.RotateCertAuthority(req); err != nil {
+  return err
+}
+```
+
+In `auto` mode, the rotation is scheduled to run in a series of phases. These phases occur in the order below, unless you set a custom schedule, or run each phase manually.
+
+```go
+// RotationPhaseStandby is the default state for a CA's rotation object - no rotation operations are underway.
+RotationPhaseStandby = "standby"
+// RotationPhaseInit is a phase of the rotation when the new certificate authority is issued, but its not used yet. It is necessary for remote trusted clusters to fetch the new certificate authority, otherwise the new clients will reject it.
+RotationPhaseInit = "init"
+// RotationPhaseUpdateClients is a phase of the rotation when client credentials will have to be updated and reloaded, but servers will use and respond with old credentials because clients (grace period).
+RotationPhaseUpdateClients = "update_clients"
+// RotationPhaseUpdateServers is a phase of the rotation when servers will have to reload and should start serving TLS and SSH certificates signed by new CA (retrieved in previous phase).
+RotationPhaseUpdateServers = "update_servers"
+// RotationPhaseRollback means that the rotation is rolling back to the old certificate authority, may occur in the case of a rotation failure.
+RotationPhaseRollback = "rollback"
+// The rotation will end in Standby after the rotation completes
+```
```

Comment:
I would format this a bit different, instead of just inlining code, list each with some english (the comment) explaining what they do.